.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 85,
+  "arrowParens": "always",
+  "semi": true,
+  "tabWidth": 2,
```

Comment:
think we've been using 4 spaces everywhere, can we bump thi sup plz?

Reply:
sorry, that was just a dumb config from a tutorial, thx for the suggestion

Reply:
updated with 4